tests/Functional/TerminusTestBase.php

```diff
@@ -418,4 +418,28 @@ protected function installTerminus2Plugins(array $plugins = [], bool $assertExit
             }
         }
     }
+
+    /**
+     * Asserts the command exists.
+     *
+     * @param string $commandName
+     *   The command name to assert.
+     */
+    protected function assertCommandExists(string $commandName)
+    {
+        $commandList = $this->terminus('list');
+        $this->assertStringContainsString($commandName, $commandList);
```

Comment:
Might this give a false positive? Maybe run `help <commandName>` and check for errors?

Reply:
@greg-1-anderson Better to ask @kporras07 since this is a code merging change (getting rid of `TerminusTestTrait` and `TerminusUtilsTrait`)